lte/gateway/python/scripts/config_stateless_agw.py

```diff
@@ -174,13 +173,16 @@ def sctpd_pre_start():
 
 
 def sctpd_post_start():
-    _start_magmad()
+    subprocess.Popen("/bin/systemctl start magma@mme".split())
+    subprocess.Popen("/bin/systemctl start magma@pipelined".split())
+    subprocess.Popen("/bin/systemctl start magma@sessiond".split())
+    subprocess.Popen("/bin/systemctl start magma@mobilityd".split())
```

Comment:
Popen does not have `PATH` awareness, so we need to pass the full path of the `service` or use `systemctl` command

Reply:
Makes sense. Can we still use subprocess.call though? Popen is preferred only when we really need to use its full set of functionality

```python3
subprocess.call("/bin/systemctl ...".split())
```

Reply:
We are using Popen here instead of subprocess since we want to make an asynchronous call to start MME service. The MME service depends on sctpd to be up and running before it can completely start. So if the `sctpd_post_start` step makes a blocking subprocess call to start MME, it will create a deadlock as both services are waiting on each other to start-up.